css/debug-bar.dev.css

```diff
@@ -344,24 +345,13 @@ body.debug-bar-maximized.debug-bar-visible {
 	border: 1px solid #c00;
 }
 
-#debug-bar-php .debug-bar-php-notice {
+#debug-bar-php .debug-bar-php-notice,
+#debug-bar-deprecated ol.debug-bar-error-notices-list li {
```

Comment:
We do for the `#debug-bar-deprecated ol.debug-bar-error-notices-list li` one.

As for the existing selector: that would be for another PR as it is unrelated to the issue being addressed here.